packages/walletconnect/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "lib",
+    "noEmit": false,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "typeRoots": ["./src/global"]
+  },
+  "include": ["src/"]
```

Comment:
Let's exclude the tests from this config so we're not adding unnecessary size to the final package